### PR TITLE
Update doc to reuse it in the Dev Guide

### PR DESCRIPTION
**Pull request by HalynaDumych**
- Move **Create platform client settings** to a separate .md file.
- Align the how-to instructions of examples with the Dev Guide
instructions.
- For links to other GitHub pages, replace MD links with HTML
(only for instructions reused in the Dev Guide).
Otherwise, they will be inactive when rendered to the doc portal.
- In doc headings, change title-style capitalization to sentence-style
to align it with the doc portal doc.

Relates-To: OLPEDGE-2348

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,23 +89,23 @@ sudo apt-get update && sudo apt-get --yes install git g++ make cmake libssl-dev
...
 
 2. (Optional) To find the required dependencies in the system, set the `OLP_SDK_BUILD_EXTERNAL_DEPS` flag to `OFF`.
 
-3. (Optional) To build the SDK as a shared library, set the `BUILD_SHARED_LIBS` flag to `ON`.
+3. (Optional) To build the Data SDK as a shared library, set the `BUILD_SHARED_LIBS` flag to `ON`.
 
 **Example**
 
-The following command builds and installs the HERE Data SDK for C++:
+The following command builds and installs the the theSDK:
```
Comment (andescu) [inline README.md:108]: Typo.

File: docs/create-platform-client-settings.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   ```cpp
+   std::chrono::seconds default_cache_expiration = std::chrono::seconds(200);
+   ```
+
+5. Set up the `OlpClientSettings` object.
+
+   ```cpp
+   olp::client::OlpClientSettings client_settings;
+   client_settings.authentication_settings = auth_settings;
+   client_settings.task_scheduler = std::move(task_scheduler);
```
[the remark this answers is not in the document]
Comment (HalynaDumych) [reply docs/create-platform-client-settings.md]: Changed.

File: docs/create-platform-client-settings.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   client_settings.network_request_handler = std::move(http_client);
```
[the remark this answers is not in the document]
Comment (HalynaDumych) [reply docs/create-platform-client-settings.md]: Changed.

File: README.md (hunk 1 above)
Comment (HalynaDumych) [reply README.md]: Modified.

**Review by andescu — DISMISSED** [review]

**Review by andescu — APPROVED** [review]

**Review by mykhailo-kuchma — APPROVED** [review]

**Pull request merged**